src/NuGet.ProjectModel/LockFileFormat.cs

```diff
@@ -177,7 +214,7 @@ private JProperty WriteTarget(LockFileTarget target)
             return new JProperty(key, json);
         }
 
-        private LockFileTarget ReadTarget(string property, JToken json)
+        private static LockFileTarget ReadTarget(string property, JToken json)
```

Comment:
I started `static`ifying `LockFileFormat` but since the VS extension depends on it, I didn't want to make the final change until @emgarten was aware and onboard :)